src/custom/state/orders/middleware.ts

```diff
@@ -102,6 +107,26 @@ export const popupMiddleware: Middleware<{}, AppState> = store => next => action
           descriptor: 'was traded'
         })
 
+        return { id, popup }
+      })
+    } else if (action.type === 'order/cancelOrdersBatch') {
+      // Why is this condition not using a `isAnyOf` like the others?
+      // For a reason that I'm not aware, if I do that the following `else`
+      // complains that it'll never be reached.
+      // If you know how to fix it, let me know.
```

Comment:
🤷 maybe @W3stside knows? 

I see it tells that the `action` type is `never` which is weird!